controllers/hyperconverged/hyperconverged_controller_test.go

```diff
@@ -32,10 +32,6 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
-	kubevirtcorev1 "kubevirt.io/api/core/v1"
```

Comment:
Please keep the kubevirt imports in a separate group than the hco imports

Reply:
As I totally agree with you, this is a semi-automated revert and I'd like it to match the commit it introduced. 

Can I keep this here and move the import in a separate PR?

Reply:
@nunnatsa FYI https://github.com/kubevirt/hyperconverged-cluster-operator/pull/2679